azure-pipelines-integration-dartlab.yml

```diff
@@ -0,0 +1,59 @@
+    testExecutionJobStrategy:
+      matrix:
+        debug_32:
+          _configuration: Debug
+    testMachineTotalCount: 4
+    testAgentElevated: true
+    preTestMachineConfigurationStepList:
+    - checkout: none
+    - download: VisualStudioBuildUnderTest
+    - task: PowerShell@2
+      name: SetProductsDropName
+      displayName: Set 'VisualStudio.BuildUnderTest.ProductsDropName'
+      inputs:
+        filePath: $(DartLab.Path)\Scripts\VisualStudio\Build\Get-VisualStudioDropName.ps1
+        arguments: -DropNamePrefix 'Products' -VstsDropUrlsJson '$(Pipeline.Workspace)\VisualStudioBuildUnderTest\BuildArtifacts\VstsDropUrls.json' -OutVariableName 'VisualStudio.BuildUnderTest.ProductsDropName'
+    deployAndRunTestsStepList:
+    - checkout: RazorMirror
+      path: $(Build.SourcesDirectory)
```

Comment:
I was wrong and a path isn't needed here. Since it is the only checkout it will pull them to the expected folder.

```suggestion
```